Foundational Supporters: Collaboration network

neural-loop opened the pull request:
This update sees the introduction of the Foundational Supporters.  On our supporters page, there is the addition of a couple tiers - which are mostly invitations for organizations to reach out to us. The foundational tier, as the 'collaboration network' underlines a shared vision towards more collaborative neuromorphics: unity through diversity. 

- [x] BindsNet https://github.com/BindsNET/bindsnet/pull/712#event-18942713971
- [x] Brian2 https://github.com/brian-team/brian2/pull/1665#event-18940082687  
- [x] Genn https://github.com/genn-team/genn/pull/699#event-18924061555
- [x] Faery https://github.com/aestream/faery/pull/30#issuecomment-3140943649 
- [x] NIR https://github.com/neuromorphs/NIR/pull/161
- [x] snnTorch https://github.com/jeshraghian/snntorch/pull/392
- [x] NIRTorch https://github.com/neuromorphs/NIRTorch/pull/47
- [x] AEStream: https://github.com/aestream/aestream/pull/117
- [ ] Sottosoglia Podcast https://www.youtube.com/watch?v=j6nGd5APAXE

Staging/Dev Preview
- Landing Page: https://neural-loop.github.io/open-neuromorphic.github.io/
- Supporters Page: https://neural-loop.github.io/open-neuromorphic.github.io/supporters/
- Draft Announcement: https://neural-loop.github.io/open-neuromorphic.github.io/blog/diverse-and-united-neuromorphic-collaboration-network/
- About page: https://neural-loop.github.io/open-neuromorphic.github.io/ (expands scope a bit to include more diverse viewpoints)

neural-loop committed:
feat: Add NIRTorch and Faery software pages

neural-loop committed:
feat: supporters - Revamp supporter tiers and display

Restructured supporters into Platinum, Gold, and Foundational tiers.
Updated supporter data and layout for new tiers.
Added supporter card component.

neural-loop committed:
feat: supporters Add supporter tiers and data

Adds supporter tiers to the supporters page.
Includes platinum, gold, and foundational tiers.
Updates foundational supporter data.

neural-loop committed:
fix(community-partners): Improve logo display

The `object-contain` class was added to the logo image to better handle logos with different aspect ratios.

neural-loop committed:
feat: content: Add faery and nirtorch software tags

neural-loop committed:
feat: Improve community partner section and bubbles

This commit enhances the community partner section with:

- Adds project names/descriptions to the partner links
- Improves bubble content and styling
- Introduces highlighting for active partner logos
- Adds a cooldown period for bubble visibility
- Updates supporter tier display logic

neural-loop committed:
feat: Style updates to community partners section

neural-loop committed:
feat: Style community partners section and hero background

Adds styling for community partners section and hero background.
Also updates typography and main workflow.

neural-loop committed:
chore: postcss.config.js: Update safelist for purged styles

neural-loop committed:
chorefeat: postcss: postcss: Add 'is-highlighted' to safelist
: Add 'is-highlighted' to safelist

neural-loop committed:
fix(shortcodes): Update supporter tier heading text

neural-loop committed:
fix(shortcodes): Update supporter tier heading text

neural-loop committed:
feat(index): Display recent event on homepage

Adds logic to fetch and display the most recent past event on the homepage.

neural-loop committed:
style: Improve background patterns & colors

neural-loop committed:
style: Improve background patterns & colors

neural-loop committed:
Chore: workflow: Remove collaboration-network branch

neural-loop committed:
chore: workflow - Add collaboration-network branch

neural-loop committed:
feat: Add snnTorch as a foundational supporter

neural-loop committed:
style: Adjust spacing on hero and partners sections

neural-loop committed:
feat: Add community resources page and Sottosoglia Podcast

Adds a new community resources page to the website.
Includes Sottosoglia Podcast as a foundational supporter.
Updates menus and adds share-nodes icon.

neural-loop committed:
feat: Add community resources page and Sottosoglia Podcast

Adds a new community resources page to the website.
Includes Sottosoglia Podcast as a foundational supporter.
Updates menus and adds share-nodes icon.

neural-loop committed:
feat: Add community resources page and Sottosoglia Podcast

Adds a new community resources page to the website.
Includes Sottosoglia Podcast as a foundational supporter.
Updates menus and adds share-nodes icon.

neural-loop committed:
feat(menus): Update navigation menus

Removed ONM Initiatives under explore-resources and added it under about.

neural-loop committed:
docs(sottosoglia-podcast): Remove mention of Italian language

neural-loop committed:
docs(sottosoglia-podcast): Remove mention of Italian language

neural-loop committed:
docs(blog): Expand on Collaboration Network details

neural-loop committed:
style: Adjust hero and partner styles

The hero section now has no bottom padding.
The partner section has a white background and bottom padding.

neural-loop committed:
refactor: Refactor community partners and rename blog

- Renamed blog post file.
- Refactored community partner styles and layout.

neural-loop committed:
feat: Improve contribution bubble appearance and behavior

The transition duration for hiding bubbles is increased and the z-index is adjusted to ensure bubbles appear above other elements. The checkered patterns within the description box have been updated to match their corresponding primary and secondary colors of the projects.

neural-loop committed:
docs(about): Update about page content

neural-loop committed:
refactor: typography-custom.scss, community-partners.html

Consolidated link styling and adjusted spacing. Removed dedicated byline-link class and incorporated it into styled-link. Updated community partners layout for consistent spacing.

neural-loop committed:
feat(layouts): Prioritize OG image for event thumbnails

neural-loop committed:
refactor(layouts): Simplify thumbnail logic

Removed OG image prioritization. Now uses front matter 'image' param or default.

DRosen766 commented:
<img width="1469" height="683" alt="Screenshot 2025-08-04 at 7 48 40 PM" src="https://github.com/user-attachments/assets/37b790d9-cb9e-47fb-8939-458930858cf8" />
Looks fantastic!
Any chance a picture could be added to the new blog post so they match here?

neural-loop committed:
feat: blog: Add ONM collaboration network post

Adds a blog post announcing the Open Neuromorphic Collaboration Network and includes image and attribution.

neural-loop commented:
>  Looks fantastic! Any chance a picture could be added to the new blog post so they match here?

<img width="1479" height="969" alt="image" src="https://github.com/user-attachments/assets/3b084977-d499-4b1a-9b29-eb01a651528a" />

Jegp reviewed (approved):
Absolutely wonderful contribution, @neural-loop. As always!

One comment relates to the different supporter levels (platinum, gold, etc.). Do we have a clear definition for who to put where? What does it take to become platinum vs. gold?

neural-loop commented:
> Absolutely wonderful contribution, @neural-loop. As always!
> 
> One comment relates to the different supporter levels (platinum, gold, etc.). Do we have a clear definition for who to put where? What does it take to become platinum vs. gold?

We don't have a clear definition, and it is currently framed to solicit leads - which could become conversations that might better help define these. Roughly for where we are at I think Platinum is sort of for orgs that contribute enough that it truly unlocks some constraints ($10k+) and Gold is below that. 

It isn't exact science at this point and platinum could be $$ or even looking at how impactful a support is (which could be more due to timing, specific resources provided, etc)
